doc/release-notes/tapyrus/release-notes-0.3.0.md

```diff
@@ -0,0 +1,64 @@
+Tapyrus version 0.3.0 is now available for download at:
+  https://github.com/chaintope/tapyrus-core/tapyrus-0.3.0/
+
+Please report bugs using the issue tracker at github:
+  https://github.com/chaintope/tapyrus-core/issues
+
+Project source code is hosted at github; you can get
+source-only tarballs/zipballs directly from there:
+  https://github.com/chaintope/tapyrus-core/tarball/v0.3.0  # .tar.gz
+
+
+How to Upgrade
+==============
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down. Follow the instruction in [getting_started](https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/getting_started.md#how-to-start-a-node-on-tapyrus-testnet) to sart a new Tapyrus node.
+
```

Comment:
The link destination is preferably the v0.3.0 branch instead of the master branch.